src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -273,12 +273,47 @@ private void testRemindActions() {
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSENT);
         homePage.goToPreviousPage(InstructorHomePage.class);
 
-        ______TS("remind action: CLOSED feedback session");
 
-        homePage.verifyUnclickable(homePage.getRemindLink(feedbackSessionClosed.getCourseId(),
-                                                          feedbackSessionClosed.getFeedbackSessionName()));
-        homePage.verifyUnclickable(homePage.getRemindOptionsLink(feedbackSessionClosed.getCourseId(),
-                                                                 feedbackSessionClosed.getFeedbackSessionName()));
+
+        ______TS("remind action: CLOSED feedback session - inner button");
+
+        homePage.clickRemindOptionsLink(feedbackSessionClosed.getCourseId(), feedbackSessionClosed.getFeedbackSessionName());
+        homePage.clickAndCancel(homePage.getRemindInnerLink(feedbackSessionClosed.getCourseId(),
+                                                            feedbackSessionOpen.getFeedbackSessionName()));
```

Comment:
@damithc  should i correct the indentation in other methods which i have not edited also like in 

             ______TS("remind action: PUBLISHED feedback session");

        homePage.verifyUnclickable(homePage.getRemindLink(feedbackSessionPublished.getCourseId(),
                                                          feedbackSessionPublished.getFeedbackSessionName()));
        homePage.verifyUnclickable(homePage.getRemindOptionsLink(feedbackSessionPublished.getCourseId(),
                                                                 feedbackSessionPublished.getFeedbackSessionName()));`